README.md

```diff
@@ -7,29 +7,310 @@
   />
 </p>
 
+# FlutterBleLib
+A library for your all your Bluetooth Low Energy needs in Flutter. Internally utilises Polidea's [MultiPlatformBleAdapter](https://github.com/Polidea/MultiPlatformBleAdapter).
 
-# FlutterBle Library
+### More from Polidea
+Check out other Polidea's BLE libraries:
+* [react-native-ble-plx](https://github.com/Polidea/react-native-ble-plx)
+* [RxAndroidBle](https://github.com/Polidea/RxAndroidBle)
+* [RxBluetoothKit](https://github.com/Polidea/RxBluetoothKit)
 
-Under construction. Stay tuned.
+## Configuration and installation
+TBD
 
-## License
+## How to use
+The library is organised around a few base entitites, which are:
+* **BleManager**
+* **Peripheral**
+* **Service**
+* **Characteristic**
+
+The basic idea is to create an instance of **BleManager**, use it to create/release native clients of the underlying libraries, scan for peripherals and then operate on **Peripheral**.
+
+### BleManager
+This entity serves as the library's entry point. It doesn't track any state, so it is safe to create as many of those as you wish.
+
+#### Initialization
+```dart
```

Comment:
I would rather show example usage instead of a signature. exactly as in https://github.com/Polidea/RxAndroidBle